### DISCUSSION TITLE
Problem with fetching React JSON Form using JS API

### DISCUSSION THREAD

**Discussion by ErikKoinberg** [bhch/django-jsonform#152] [Q&A] [answered]
The code is like this:
```
var formEl = document.getElementById('id_structure_jsonform'); // "object"
console.log(typeof(reactJsonForm)) // "object"
var form = reactJsonForm.getFormInstance('id_structure_jsonform'); // "undefined"
```

I have imported the libraries, and the reactJsonForm variable is defined. Furthermore, I can confirm that the field structure exists and I can also find it using document.getElementById(). 

What could be the reason that the form is not fetched here?

**Comment by bhch** [comment]
Please upload a sample repo on github so I can reproduce it.

**Comment by ErikKoinberg** [comment, marked as answer]
I was able to fetch it after calling the code from inside 
`$(document).ready()`
using jQuery. I guess other ways of accessing the content after it is loaded should also work
